commands/log.js

```diff
@@ -1,87 +1,54 @@
         const server = Math.floor(Number(args[0]));
-        let size = Math.floor(Number(args[1]));
-        let sl = Math.floor(Number(args[2]));
-
-        if (isNaN(size)) {
-            msg.reply(`Please give the amount of lines you want`)
-                .catch((err) => {internal_error(err); return})
-        }
-
+        let size = args[1];
         let sizeLimit = 50;
         let defaultSize = 10;
-        let defaultSl = 1;
 
         if (!server) {
             msg.channel.send('Please pick a server first. Just the number (1-8)')
-                .catch((err) => {internal_error(err); return});
+                .catch((err) => { internal_error(err); return });
             return;
         }
-        
         if (!size) {
             size = defaultSize;
             msg.channel.send(`Using standard amount of lines (${defaultSize}):`)
-                .catch((err) => {internal_error(err); return});
+                .catch((err) => { internal_error(err); return });
         } else if (size > sizeLimit) {
             size = defaultSize;
-            msg.channel.send(`Cannot get more than ${sizeLimit} lines, will get ${defaultSize} instead`)
-                .catch((err) => {internal_error(err); return});
+            msg.channel.send(`Cannot get more than 50 lines, will get ${defaultSize} instead`)
+                .catch((err) => { internal_error(err); return });
         } else if (size <= 0) {
             size = defaultSize;
             msg.channel.send(`Cannot be negative or 0, using standard amount of lines (${defaultSize}):`)
-                .catch((err) => {internal_error(err); return});
-        }
-        
-        if (!sl || isNaN(sl)) {
-            sl = defaultSl;
-        } else if (sl > size) {
-            sl = defaultSl;
-            msg.channel.send(`Starting line cannot be larger than ending line, will use default starting line ${defaultSl} instead`)
-                .catch((err) => {internal_error(err); return});
-        } else if (sl <= 0) {
-            sl = defaultSl;
-            msg.channel.send(`Cannot be negative or 0, using standard starting line (${defaultSl}):`)
-                .catch((err) => {internal_error(err); return});
+                .catch((err) => { internal_error(err); return });
         }
-
         if (server < 9 && server > 0) {
             console.log(`Server is ${server}`);
-            get_logs(server, size, sl, msg, internal_error)
-                .catch((err) => {internal_error(err); return})
+            get_logs(server, size, msg)
+                .catch((err) => { internal_error(err); return })
         } else {
-            msg.reply(`Please pick a server first. Just the number (currently 1-8). Correct usage is \` .exp log <server#> <#lines>\``)
-                .catch((err) => {internal_error(err); return})
+            msg.reply(`Please pick a server first. Just the number (currently 1-8). Correct usage is \` .exp unjail <server#> <username>\``)
```

Comment:
```suggestion
            msg.reply(`Please pick a server first. Just the number (currently 1-8). Correct usage is \` .exp log <server#> <#oflines> <optionalStartingLine#>\``)
```
Not sure why we reverted *back* to jail but :shrug:

Reply:
lol